### PR TITLE
Refactoring Default textarea size

### PR DESCRIPTION
**Pull request by WulfPlasma**
@igeligel Let me know if I need to change anything or do anything to improve the code. Thank you for letting me work on this.

### CODE REVIEW

**Comment by WulfPlasma** [comment]
#71

**Comment by WulfPlasma** [comment]
I see it has failed, let's see if I can fix that.

**Comment by WulfPlasma** [comment]
Sorry if I did anything wrong, it's been awhile since I've contributed to open source and I've only done a few PRs

**Comment by igeligel** [comment]
Hey @WulfPlasma, no problem at all. I am wondering what kind of setup are you running? Like which Editor or IDE and how do you make commits? It seems like something broke the `index.tsx` file.

**Comment by WulfPlasma** [comment]
I currently use atom for any Github related work

File: src/index.tsx [hunk 1]
```diff
@@ -119,7 +124,8 @@ export interface Props extends HTMLAttributes<HTMLDivElement> {
   maxContentLength?: number;
 }
 
-export const InOutTextarea: FC<Props> = props => {
+export const InOutTextarea: FC<Props> = (props) => {
+  return
```
Comment (igeligel) [inline src/index.tsx:128]: This line should be removed I believe

File: src/index.tsx [hunk 2]
```diff
@@ -266,7 +271,7 @@ export const InOutTextarea: FC<Props> = props => {
             <IconX size={32} />
           </IconContainer>
         </TextAreaContent>
-        <TextAreaContent>
```
Comment (igeligel) [inline src/index.tsx:269]: Can you revert that?

File: src/index.tsx [hunk 3]
```diff
@@ -283,5 +288,7 @@ export const InOutTextarea: FC<Props> = props => {
         </TextAreaContent>
       </ConvertCardContent>
     </ConvertCard>
+    </TextAreaContent>
+    </>
```
Comment (igeligel) [inline src/index.tsx:292]: These two lines have to be deleted

**Comment by igeligel** [comment]
Ahh alright, @WulfPlasma I made some comments on the code. Maybe you can change them. You might want to add the addon: https://atom.io/packages/language-typescript-react to your editor because this enabled proper support for this setup I believe. Before you make a commit run the command `yarn lint --fix`. That should fix most of the issues.

**Comment by WulfPlasma** [comment]
@igeligel after running `yarn lint --fix` and using `git status` it says all of those files were modified, should it have done this? I fixed the errors in the code though, so I'll make a commit now.
Screenshot 2020-10-14 130137

**Comment by igeligel** [comment]
Nope, it should not. It seems like something with the prettier config might be not right. I will have a detailed look tomorrow.

**Comment by WulfPlasma** [comment]
Meanwhile I'll try to find a fix, if I'm able to fix it and not get any errors I'll make a commit. Thank you for the help.

**Comment by WulfPlasma** [comment]
@igeligel any updates?

**Comment by WulfPlasma** [comment]
I’ll keep the branch just incase we find a fix, for now I’ll close the PR and come back to it later if need be.

**Pull request closed without merging**